packages/sources/cfbenchmarks/src/transport/birc.ts

```diff
@@ -93,8 +93,8 @@ export const transport = new CfBenchmarksBIRCTransport({
       const key = param.tenor as VALID_TENORS
       const value = Number(latestUpdate.tenors[key])
 
-      if (!latestUpdateIsCurrentDay(latestUpdate.time)) {
-        const warning = 'Latest update from response is not in current day'
+      if (!latestUpdateIsWithinLast24h(latestUpdate.time)) {
+        const warning = 'Latest update from response is not in current day %o'
         logger.warn(warning, { latestUpdate })
```

Comment:
Minor: I'd find the below more readable, feel free to take it or leave it. 

```suggestion
        logger.warn(`Latest update from response is not in current day ${latestUpdate}`)
```

Reply:
The log actually ends up looking like this instead of with the expanded object 😢 
`Latest update from response is not in current day [object Object]`

Reply:
I'm guessing it implicitly runs `JSON.stringify()` if you see the full object when you use %o.